src/app/data/data-types.ts

```diff
@@ -2,4 +2,14 @@ export interface Character {
     name: string;
     race: string;
     class: string;
+}
+
+export interface Race { //TODO: add race details
```

Comment:
ToDos in code are often forgotten, better if you create an issue on the project to fix this and link to it here

Reply:
Adding the descriptions will cover what I wanted to do here so I'll remove it.